geode-core/src/main/java/org/apache/geode/cache/control/RebalanceResults.java

```diff
@@ -99,4 +99,11 @@
    * @return the total time, in milliseconds, spent transferring primaries
    */
   long getTotalPrimaryTransferTime();
+
+  /**
+   * Returns the total number of members on which command is executed.
+   *
+   * @return the total number of members on which command is executed
+   */
+  int getTotalNumOfMembers();
```

Comment:
This method might be better named "getTotalMembersExecutedOn", or something else that makes it clear that it's specifically returning the number of members the command was executed on.